go/noise/gaussian_noise.go

```diff
@@ -155,6 +186,23 @@ func addGaussian(x, sigma float64) float64 {
 	return roundToMultipleOfPowerOfTwo(x, granularity) + float64(sample)*granularity
 }
 
+// getConfidenceIntervalGaussian computes the confidence interval using the inverse error function.
+func getConfidenceIntervalGaussian(noisedValue, confidenceLevel, sigma float64) ConfidenceInterval {
+	shiftingValue := inverseCDFGaussian(sigma, confidenceLevel)
```

Comment:
inverseCDFGaussian should be either called on (sigma, 1 - alpha / 2) or (sigma, alpha / 2). in the latter case (which is what I would suggest to do), you need to add the result noisedX for the lower bound and subtract it from the upperBound (because the result will be negative).